databricks-dbutils-scala/src/test/scala/com/databricks/sdk/scala/dbutils/integration/SdkTaskValuesIntegrationTest.scala

```diff
@@ -0,0 +1,76 @@
+package com.databricks.sdk.scala.dbutils.integration
+
+import com.databricks.sdk.scala.dbutils.CommandContext
+import org.scalatest.matchers.should.Matchers.convertToAnyShouldWrapper
+import org.scalatest.matchers.should.Matchers.be
+
+class SdkTaskValuesIntegrationTest extends IntegrationTestBase {
```

Comment:
This doesn't need to be an integration test since none of these tests run in DBR or interact with the Databricks REST API. Instead, let's move it up one directory into the `com.databricks.sdk.scala.dbutils` package and extend AnyFlatSpec like ProxyDbfsTest.